packages/utils/index.js

```diff
@@ -8,3 +8,10 @@ export { formatPrice } from './localization/CurrencyFormatter';
 export type { Price } from './localization/CurrencyFormatter';
 export { default as DateFormatter } from './localization/DateFormatter';
 export { default as DateUtils } from './localization/DateUtils';
+
+// src
+
+export { default as withContext } from './src/withContext/withContext';
+
+// other
+export const noop = () => {};
```

Comment:
Is that really necessary?

Reply:
Well, I expect we will have more contexts, and instead of defining this once in every context, I think it is nice to just import it.